src/RF24LogLevel.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * RF24LogLevel.h
+ *
+ *  Created on: 5 paź 2020
+ *      Author: wmarkowski
+ */
+
+#ifndef SRC_RF24LOGLEVEL_H_
+#define SRC_RF24LOGLEVEL_H_
+
+#include <avr/pgmspace.h>
+
+enum RF24LogLevel : uint8_t
+{
+   ERROR = 0, WARN, INFO, DEBUG, TRACE

```

Comment:
I'm not quite sure if **TRACE** log level is really needed here. Maybe just haven four of log levels is good enough. IIf we remove TRACE then we can save a bit of memory. I will post memory usage soon.

Reply:
I don't think enum values are kept if not used so it probably isn't necessary to remove it.

Reply:
You mean that compiler can make optimalisation and remove unused enum values? That make sense.

What I meant was: if we give the user a chance to use TRACE, then somebody may use it in his project (or even for example in some other library). Then this TRACE will be compiled and will consume some memory. Ah, there is also a trace method in RF24Logger.h which will be always compiled and currently it takes 0x3e bytes of FLASH.

Reply:
I don't think it's a massive concern for now. 

It would be nice to be able to `#define` a maximum allowed verbosity, that would remove/nullify all the too verbose logging calls.

Reply:
>It would be nice to be able to #define a maximum allowed verbosity, that would remove/nullify all the too verbose logging calls.

That would be easily done with the help of sublevels (proposed by @2bndy5). I will take a closer look at this.

Reply:
The maximum allowed verbosity is now available as a number between 0 (log OFF) and 255 (log ALL). Just call ```RF24LogHandler.setLogLevel(uint8_t)```